socorro/tests/processor/rules/test_mozilla.py

```diff
@@ -1439,6 +1440,107 @@ def test_missing_crashing_thread(self, tmp_path):
         assert "topmost_filenames" not in processed_crash
 
 
+def generate_debug_id():
+    """Generates a valid debug id."""
+
+
```

Comment:
this isn't used, doesn't appear to do anything, and looks like it was replaced by `debug_id_helper.generate()`
```suggestion
```

Reply:
You're right! I wrote it and then decided to do a pytest fixture because I could use it elsewhere. I'll remove it.